packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -782,6 +783,21 @@ class RawScrollbar extends StatefulWidget {
   /// {@endtemplate}
   final ScrollNotificationPredicate notificationPredicate;
 
+  /// {@template flutter.widgets.Scrollbar.interactive}
```

Comment:
This doc should link to enableGestures and enableGestures should link to the interactive property. However, maybe it would better to leave this property out and have subclasses add the property and override enableGestures.

Reply:
I am not sure I follow. This is similar to the `isAlwaysShown`/`showScrollbar` so I tried to follow to same pattern there.

Reply:
It makes sense to link this doc to enableGestures (and vice versa) since enableGestures depends on this property by default. 

You could eliminate it altogether, since CupertinoScrollbar doesn't need it, and MaterialScrollbar could add it.

Reply:
Ah thanks, I see. I think it's ok for `RawScrollbar` to have `interactive`. Some folks really do want just a passive indicator, I think it makes sense for `RawScrollbar` to have the option. I'll update the docs. :)